ports/mimxrt10xx/boards/feather_m7_1011/board.c

```diff
@@ -27,8 +27,26 @@
 
 #include "boards/board.h"
 #include "mpconfigboard.h"
+#include "shared-bindings/microcontroller/Pin.h"
 
 void board_init(void) {
+    // SWD Pins
+    common_hal_never_reset_pin(&pin_GPIO_AD_13);//SWDIO
+    common_hal_never_reset_pin(&pin_GPIO_AD_12);//SWCLK
+    common_hal_never_reset_pin(&pin_GPIO_AD_09);//SWO
+
+    // FLEX flash
+    common_hal_never_reset_pin(&pin_GPIO_SD_12);
+    common_hal_never_reset_pin(&pin_GPIO_SD_11);
+    common_hal_never_reset_pin(&pin_GPIO_SD_10);
+    common_hal_never_reset_pin(&pin_GPIO_SD_09);
+    common_hal_never_reset_pin(&pin_GPIO_SD_08);
+    common_hal_never_reset_pin(&pin_GPIO_SD_07);
+    common_hal_never_reset_pin(&pin_GPIO_SD_06);
+
+    // USB Pins
+    common_hal_never_reset_pin(&pin_GPIO_12);
+    common_hal_never_reset_pin(&pin_GPIO_13);
```

Comment:
These pins are not used for their USB function on this Feather, this can be removed.

Reply:
How is this differentiated between the EVK and the feathers? I found that if I reset these pins on the EVK, it would mess up USB function.

Reply:
It's because on the EVK the OTG port can be used for both Device and Host modes so the ID and OC pins are actually used by the USB power switch. On the Feathers we only support Device so those pins are not used for any USB-related functionality.

Reply:
Do you pass this information to tinyUSB over a CFLAG or no?